packages/toast/src/InternalToast/defaultProps.ts

```diff
@@ -0,0 +1,10 @@
+import { ToastProps, Variant } from '../Toast.types';
+
+export const defaultToastProps: Required<
+  Pick<ToastProps, 'variant' | 'progress' | 'dismissible' | 'timeout'>
+> = {
+  variant: Variant.Note,
+  progress: 1.0,
+  timeout: 6_000,
+  dismissible: true,
+};
```

Comment:
nit: I assume these are defaultProps for just the story file? It'd be nice if that was indicated.